packages/apps/stock/src/index.ts

```diff
@@ -99,30 +99,15 @@ export const StockModule: Module = {
   },
 };
 
-export * from './components';
-
-export {
-  displayInventorySeq,
-  displayItemTrackingNumber,
-  displayPartner,
-  displayStockMoveSeq,
-} from './utils/displayers';
-export {
-  filterInventoryByRef,
-  filterItemByStockMoveSeq,
-  filterPartner,
-  filterProduct,
-  filterStockLocation,
-  filterTrackingNumber,
-} from './utils/filters';
-
-export * from './types';
 export * from './api';
+export * from './components';
 export * from './features/asyncFunctions-index';
-
+export * from './hooks';
 export * from './screens/customerDeliveries';
 export * from './screens/internalMoves';
 export * from './screens/inventories';
 export * from './screens/products';
 export * from './screens/stockCorrections';
 export * from './screens/supplierArrivals';
```

Comment:
Those exports are not working 🙈 The index file in screen folder is useful for the navigator in core package but each package should export the screen components to help external package to re-use it.

You can just modify index file in screen folders to export the default export for navigation AND each component one by one.